internal/commands/render.go

```diff
@@ -75,11 +76,14 @@ func prepareCustomAction(actionName string, dockerCli command.Cli, appname strin
 	if err != nil {
 		return nil, nil, nil, err
 	}
-	bundle, ref, err := cnab.ResolveBundle(dockerCli, bundleStore, appname)
+	bundle, ref, err := cnab.GetBundle(dockerCli, bundleStore, appname)
 	if err != nil {
+		if strings.HasSuffix(appname, ".dockerapp") {
```

Comment:
Maybe this test should be moved to `cnab.GetBundle` ? So all the commands would have the same behavior?

Reply:
If your application is `toto.dockerapp` the GetBundle should return a bundle and triggers no error but I'm also wondering if having that specific warning for `.dockerapp` is useful